en/patterns/grid.md

```diff
@@ -12,7 +12,7 @@ The Grid Pattern with Column Moving, Summaries and Paging is shown below:
 
 <img class="responsive-img" src="../images/grid.png" srcset="../images/grid@2x.png 2x" />
 
-List of all the Grid functionality:
+List of all the Grid functionalities:
```

Comment:
@sdimchevski Should we align this functionalities list in the Grid / Hierarchical Grid / Tree Grid Pattern pages? Currently on the Grid and Hierarchical Grid pages it looks the same way, and on the Tree Grid page we have slight difference + links.